lib/pages/other_detail/other_detail_page.dart

```diff
@@ -5,14 +5,26 @@ import 'package:stimmungsringeapp/data/detail_pages.dart';
 import 'package:stimmungsringeapp/global_constants.dart';
 import 'package:stimmungsringeapp/pages/dashboard/bloc/bloc.dart';
 import 'package:stimmungsringeapp/pages/other_detail/bloc/bloc.dart';
-import 'package:stimmungsringeapp/repositories/dashboard_repository.dart';
 import 'package:stimmungsringeapp/widgets/avatar_row.dart';
 import 'package:stimmungsringeapp/widgets/loading_spinner_widget.dart';
 
-class OtherDetailPage extends StatelessWidget {
-  final DashboardRepository dashboardRepository;
+class OtherDetailPage extends StatefulWidget {
+  final String otherUserId;
+
+  OtherDetailPage({this.otherUserId});
+
+  @override
+  _OtherDetailPageState createState() => _OtherDetailPageState();
+}
+
+class _OtherDetailPageState extends State<OtherDetailPage> {
+  @override
+  void didChangeDependencies() {
```

Comment:
could you explain this?

Reply:
see the reso link - he recommends using a Stateful widget to kick off first event